modules/microshift-cni.adoc

```diff
@@ -0,0 +1,304 @@
+
+== Secondary gateway interface
+You can set up one additional host interface for cluster ingress and egress in the `ovn.yaml` config file.
+The additional interface will be added in a second OVS bridge br-ex1. Cluster pod traffic directed to the additional host subnet will be routed through br-ex1.
+
+//FIXME: need updated config procedure for customers that will persist across reboots
+
+//=== Manually configuring OVS bridge br-ex
+//.Procedure
+//Manually configure the OVS bridge br-ex by running the following commands.
+
+//* Initiate OVS:
+//+
+//[source, terminal]
+//----
+//$ sudo systemctl enable openvswitch --now
+//----
+//* Add the network bridge:
+//+
+//[source, terminal]
+//----
+//$ sudo ovs-vsctl add-br br-ex
+//----
+//* Add the interface to the network bridge:
+//+
+//[source, terminal]
+//----
+//$ sudo ovs-vsctl add-port br-ex <physical-interface-name>
+//----
+//The `<physical-interface-name>` is the network interface name where the node IP address is assigned.
+//* Get the bridge up and running:
+//+
+//[source, terminal]
+//----
+//$ sudo ip link set br-ex up
+//----
+//* After `br-ex up` is running, assign the node IP address to `br-ex` bridge:
+//[source, terminal]
+//----
+//$ sudo ...
+//----
+//[NOTE]
+//Adding a physical interface to `br-ex` bridge will disconnect the ssh connection on the node IP address.
+
+== Firewalld
+Explicitly allow the following OVN-Kubernetes traffic when the firewalld service is running:
+//Q: procedure?
+CNI pod to CNI pod::
+CNI pod to Host-Network pod
+Host-Network pod to Host-Network pod
+
+CNI pod::
+The Kubernetes pod that uses the CNI network
+
+Host-Network pod::
+The Kubernetes pod that uses host network
+
+=== Configuring firewalld
+.Procedure
+Insert the following firewall rules to allow the OVN-Kubernetes traffic by running the following commands.
+//Q: assumes configuring from kickstart
+* Insert the IP range:
+[source, terminal]
+----
+$ sudo firewall-offline-cmd --permanent --zone=trusted --add-source=10.42.0.0/16
+----
+
+* Insert the DNS server:
+[source, terminal]
+----
+$ sudo firewall-offline-cmd --permanent --zone=trusted --add-source=169.254.169.1
+----
+
+* Reload the firewall rules:
++
+[source, terminal]
+----
+$ sudo firewall-offline-cmd
+----
+
+== Known issue
+To avoid breaking traffic flows with a firewall restart, execute firewall commands before starting OVN-Kubernetes pods. OVN-Kubernetes makes use of iptable rules for some traffic flows (such as nodePort service). The iptable rules are generated and inserted by the `ovnkube-master` container, but are deleted when the firewall restarts. The absence of the iptable rules breaks traffic flows. If firewall commands have to be executed after OVN-Kubernetes pods have started, manually restart the `ovnkube-master` pod to trigger a reconciliation of the iptable rules.
+//See Troubleshooting for a detailed procedure. Need hard link to troubleshooting section
+
+== IP forward
```

Comment:
@mangelajo would we want to call this as a known issue or maybe just a note/limitation?

Reply:
We have the full procedure in the Troubleshooting section for this. Will the iptable be fixed at some point to persist?